### DISCUSSION TITLE
希望能够区分收发图片的类型

### DISCUSSION THREAD

**Discussion by A-kirami** [Mrs4s/go-cqhttp#915] [Ideas]
目前在手机QQ和PCQQ上，发送的表情和普通的图片是区分开的

<img src="https://i.loli.net/2021/06/10/S1D5H8GBVyLKtvq.jpg" width="40%"><img src="https://i.loli.net/2021/06/10/nSyVHBkbQ549DJP.jpg" width="40%">

能否使得gocqhttp的CQ码也能支持区分两种图片来源呢

**Comment by xiguasamall** [comment]
所以 到底是咋区分的呢
